### ISSUE TITLE
Charging Status

### ISSUE THREAD

**Issue by nottledim** [bimmerconnected/bimmer_connected#454]
### Describe the issue

Is the charging status attribute correct?

```
car: i3 120
vin: xxx
charging_status: ChargingState.CHARGING
attribution: Data provided by MyBMW
device_class: battery_charging
icon: mdi:ev-station
friendly_name: i3 120 charging_status
```

### Expected behavior

I would expect the status to be a simple value like "CHARGING"

### Which Home Assistant version are you using?

Core 2022.6.1

### What is your region?

Rest of world

### MyBMW website

- [X] I can still successfully login to the BMW MyBMW website and the car status is available there.
- [X] I have MyBMW enabled for my vehicle.

### Number of cars

- [X] I have 2 or more cars linked to the MyBMW account.
- [ ] I have a Mini vehicle linked to my account.

**Comment by rikroe** [comment]
Thanks for reporting, don't have an EV so don't see it.
Will be removed (probably with HA 2022.8), as there is the `sensor.charging_status` which gives exactly the same information directly as sensor value.

**Comment by rikroe** [comment]
Closing as implemented with 2022.9.
